bin/kube_overrides.rb

```diff
@@ -37,6 +43,28 @@
         env['valueFrom']['secretKeyRef']['name'] = current_secrets_name
       end
     end
+    overrides.each do |k, v|
```

Comment:
Why is this code inside the `container['env'].each do |env|` loop?

For the `Array` case it doesn't matter, as it will delete and add-back the override over and over, but for a `Hash` it would add the override multiple times, once for each `env` entry.

Reply:
> it would add the override multiple times

Well no, it wouldn't. So it makes no difference, after all, but I still would like to see the code pulled out of the loop. 😄

Reply:
Fixed in 7b3c931f0c7ec9d0f15e7fa29968b6b2a1baaa98 because it's just kind of embarrassing :smile: